src/applications/static-pages/createCoronavirusChatbot.js

```diff
@@ -0,0 +1,198 @@
+    if (tokenPayload.directLineURI) {
+      domain = `https://${tokenPayload.directLineURI}/v3/directline`;
+    }
+    const botConnection = window.WebChat.createDirectLine({
+      token: tokenPayload.connectorToken,
+      domain,
+    });
+    const styleOptions = {
+      hideSendBox: true,
+      botAvatarInitials: 'VA',
+      userAvatarInitials: 'You',
+      backgroundColor: '#F8F8F8',
+      primaryFont: 'Source Sans Pro, sans-serif',
+    };
+
+    const store = window.WebChat.createStore(
+      {},
+      store =>
+        function(next) {
+          return function(action) {
+            if (action.type === 'DIRECT_LINE/CONNECT_FULFILLED') {
+              store.dispatch({
+                type: 'DIRECT_LINE/POST_ACTIVITY',
+                meta: { method: 'keyboard' },
+                payload: {
+                  activity: {
+                    type: 'invoke',
+                    name: 'InitConversation',
+                    locale: user.locale,
+                    value: {
+                      // must use for authenticated conversation.
+                      jsonWebToken,
+
+                      // Use the following activity to proactively invoke a bot scenario
+
+                      triggeredScenario: {
+                        trigger: chatBotScenario,
+                      },
+                    },
+                  },
+                },
+              });
+            } else if (action.type === 'DIRECT_LINE/INCOMING_ACTIVITY') {
+              if (
+                action.payload &&
+                action.payload.activity &&
+                action.payload.activity.type === 'event' &&
+                action.payload.activity.name === 'ShareLocationEvent'
+              ) {
+                // share
+                getUserLocation(location => {
+                  store.dispatch({
+                    type: 'WEB_CHAT/SEND_POST_BACK',
+                    payload: { value: JSON.stringify(location) },
+                  });
+                });
+              }
+              setTimeout(() => {
+                document.querySelector(
+                  'div.css-y1c0xs',
+                ).scrollTop = document.querySelector(
+                  'div.css-y1c0xs',
+                ).scrollHeight;
+              });
+            }
+            return next(action);
+          };
+        },
+    );
+    const webchatOptions = {
+      directLine: botConnection,
+      styleOptions,
+      store,
+      userID: user.id,
+      username: user.name,
+      locale: user.locale,
+    };
+    startChat(user, webchatOptions);
+  }
+
+  function requestChatBot(loc) {
+    const params = new URLSearchParams(location.search);
+    const locale = params.has('locale')
+      ? extractLocale(params.get('locale'))
+      : defaultLocale;
+    const oReq = new XMLHttpRequest();
+    oReq.addEventListener('load', initBotConversation);
+    let path = `https://va-covid19-chatbot-dev.azurewebsites.net/chatBot?locale=${locale}`;
```

Comment:
I think this service needs its CORS policy to allow `localhost:3001` and `*.va.gov`.